src/client/components/widgets/GoDirectoryInput/FilterDrawer/index.tsx

```diff
@@ -0,0 +1,107 @@
+import React, {FunctionComponent} from 'react'
+import {
+    Typography,
+    createStyles,
+    makeStyles,
+    Divider,
+    Drawer,
+    Collapse,
+    Paper,
+    Grid,
+} from '@material-ui/core'
+import SortButton from '../../SortPanel/SortButton'
+
+type FilterDrawerProps = {
+    onClick: (email:boolean) => void
+    selected: boolean
+    isFilterOpen: boolean
+    isMobileView: boolean
+    setIsFilterOpen: (isFilterOpen: boolean) => void
+}
+
+const useStyles = makeStyles((theme) =>
+  createStyles({
+    filterPanel: {
+        padding: theme.spacing(1.5),
+        [theme.breakpoints.up('md')]: {
+            width: 300,
+          },
+    },
+    headerText: {
+        padding: theme.spacing(3.5),
+    },
+  })
+)
+
+// Filter the search by keyword or email
+const FilterDrawer : FunctionComponent<FilterDrawerProps> = ({
+    onClick,
+    selected,
+    isFilterOpen,
+    isMobileView,
+    setIsFilterOpen,
+}: FilterDrawerProps) => {
+    const classes = useStyles()
+
+    const handleDrawerClose = () => {
+        setIsFilterOpen(false)
+    }
+
+    return (
+        <>
+        {isMobileView? 
+        <Drawer 
+            anchor='bottom'
+            open={isFilterOpen}
+            onBackdropClick={handleDrawerClose}
+            onEscapeKeyDown={handleDrawerClose}
+            >
+            <Collapse
+                in={isFilterOpen}>
```

Comment:
i find it strange that when doing html-like code in tsx/jsx files it uses 4 spaces when our entire codebase is using 2 space indentation